h/static/scripts/directive/test/selection-tabs-test.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+var angular = require('angular');
+
+var util = require('./util');
+
+describe('selectionTabs', function () {
+  before(function () {
+    angular.module('app', [])
+      .directive('selectionTabs', require('../selection-tabs'));
+  });
+
+  beforeEach(function () {
+    angular.mock.module('app');
```

Comment:
There is a shortcut you can use here to make injecting fake dependencies easier. `angular.mock.module` takes an optional second argument which is an object mapping dependency names to fakes.

```js
var fakeAnnotationUI = {};
angular.mock.module('app', {
  annotationUI: fakeAnnotationUI,
});
```